common/src/main/java/com/example/android/uamp/media/library/AlbumArtContentProvider.kt

```diff
@@ -99,3 +90,5 @@ class AlbumArtContentProvider : ContentProvider() {
     override fun getType(uri: Uri): String? = null
 
 }
+
+const val DOWNLOAD_TIMEOUT_SECONDS = 30L
```

Comment:
Consider moving this to the top of the file (and adding a comment, maybe? I can't decide.)